modules/configuring-external-databases/proc-configuring-default-configuration-for-secrets-and-pvcs.adoc

```diff
@@ -0,0 +1,71 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-default-configuration-for-secrets-and-pvcs_{context}"]
+= Configuring default configuration for Secrets and PVCs
+
+By default, the mount path is the {product-short} container's working directory, and if you do not define the mount path, it defaults to `/opt/app-root/src`. The default configuration files are `secret-files` and PVCs. You can add the `rhdh.redhat.com/mount-path` annotation to your configuration file to specify the mount path.
+
+.Procedure
+
+. Specify the mount path where the PVC mounts to by adding the `rhdh.redhat.com/mount-path` annotation to your configuration as shown in the following example:
++
+.Example specifying where the PVC mounts
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: _<my_claim>_
+  annotations:
+    rhdh.redhat.com/mount-path: /mount/path/from/annotation
+----
+where:
+
+`rhdh.redhat.com/mount-path`:: Specifies which mount path the PVC mounts to (in this case, `/mount/path/from/annotation` directory).
+<my_claim>:: Specifies the PVC to mount.
+
+. Specify the mount path where the Secret mounts to by adding the `rhdh.redhat.com/mount-path` annotation to your configuration as shown in the following example:
++
+.Example specifying where the Secret mounts
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: _<my_secret>_
+  annotations:
+    rhdh.redhat.com/mount-path: /mount/path/from/annotation
+----
+where:
+
+<my_secret>:: Specifies the Secret name.
+
+. Use the following configuration to mount Secrets to all containers:
++
+.Example configuration for mounting to all containers
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: _<my_secret>_
+  annotations:
+    rhdh.redhat.com/containers: `*`
+----
++
```

Comment:
There are 3 options:
- no annotation for only backstage-backend container
 - "*"- to mount to all containers (as mentioned)
 - comma separated list of containers to mount (as mentioned)
I do not think one option is more important than another, maybe it makes sense just to list them (not saying Important and optional)?